data/names/ru.json

```diff
@@ -5870,7 +5870,624 @@
       "Янсен",
       "Янси",
       "Ян",
-      "Ярбро"
+      "Ярбро",
+      [ "Авель", "Abel" ],
+      [ "Авраам", "Abraham" ],
+      [ "Акерс", "Akers" ],
+      [ "Аланиз", "Alaniz" ],
+      [ "Александр", "Alexander" ],
+      [ "Алли", "Alley" ],
+      [ "Диамант", "Diamond" ],
+      [ "Амайя", "Amaya" ],
+      [ "Амарал", "Amaral" ],
+      [ "Амос", "Amos" ],
+      [ "Андрей", "Andrew" ],
+      [ "Арии", "Arias" ],
+      [ "Аруда", "Arruda" ],
+      [ "Арфа", "Harp" ],
+      [ "Байнум", "Bynum" ],
+      [ "Бэкон", "Bacon" ],
+      [ "Бальдерас", "Balderas" ],
+      [ "Банкс", "Banks" ],
+      [ "Барретт", "Barrett" ],
+      [ "Барриентос", "Barrientos" ],
+      [ "Барфилд", "Barfield" ],
+      [ "Бас", "Bass" ],
+      [ "Бассетт", "Bassett" ],
+      [ "Баумана", "Bauman" ],
+      [ "Бегай", "Begay" ],
+      [ "Бейн", "Bain" ],
+      [ "Беланжер", "Belanger" ],
+      [ "Уайт", "White" ],
+      [ "Бердик", "Burdick" ],
+      [ "Берк", "Burk" ],
+      [ "Беркетт", "Burkett" ],
+      [ "Беркс", "Burks" ],
+      [ "Беркхарт", "Burkhart" ],
+      [ "Бернар", "Bernard" ],
+      [ "Бернс", "Burns" ],
+      [ "Бернье", "Bernier" ],
+      [ "Берр", "Burr" ],
+      [ "Беррис", "Burris" ],
+      [ "Берт", "Burt" ],
+      [ "Бертон", "Burton" ],
+      [ "Биби", "Beebe" ],
+      [ "Бизли", "Beasley" ],
+      [ "Бёрч", "Birch" ],
+      [ "Бланд", "Bland" ],
+      [ "Блаунт", "Blount" ],
+      [ "Блек", "Black" ],
+      [ "Блидсо", "Bledsoe" ],
+      [ "Блэкбёрн", "Blackburn" ],
+      [ "Бин", "Bean" ],
+      [ "Баттл", "Battle" ],
+      [ "Боннэр", "Bonner" ],
+      [ "Борхес", "Borges" ],
+      [ "Ботельо", "Botelho" ],
+      [ "Боуг", "Baugh" ],
+      [ "Бошан", "Beauchamp" ],
+      [ "Брайс", "Brice" ],
+      [ "Брантли", "Brantley" ],
+      [ "Бранхам", "Branham" ],
+      [ "Брасвелл", "Braswell" ],
+      [ "Брей", "Bray" ],
+      [ "Брейди", "Brady" ],
+      [ "Бренд", "Brand" ],
+      [ "Брубейкер", "Brubaker" ],
+      [ "Бруссар", "Broussard" ],
+      [ "Брэннон", "Brannon" ],
+      [ "Бургер", "Burger" ],
+      [ "Бурлесон", "Burleson" ],
+      [ "Куик", "Quick" ],
+      [ "Бьюли", "Beaulieu" ],
+      [ "Бэбб", "Babb" ],
+      [ "Бэрри", "Berry" ],
+      [ "Бэррон", "Barron" ],
+      [ "Бёрден", "Burden" ],
+      [ "Бёрдетт", "Burdette" ],
+      [ "Бёрнем", "Burnham" ],
+      [ "Вайз", "Wise" ],
+      [ "Вайзмен", "Wiseman" ],
+      [ "Валентайн", "Valentine" ],
+      [ "Ванхорн", "Vanhorn" ],
+      [ "Ватсон", "Watson" ],
+      [ "Ватт", "Watt" ],
+      [ "Вейр", "Ware" ],
+      [ "Джолли", "Jolly" ],
+      [ "Хэм", "Ham" ],
+      [ "Виггинс", "Wiggins" ],
+      [ "Видаль", "Vidal" ],
+      [ "Виджид", "Vigil" ],
+      [ "Виейра", "Vieira" ],
+      [ "Вишневский", "Wisniewski" ],
+      [ "Дэнт", "Dent" ],
+      [ "Волк", "Wolf" ],
```

Comment:
I skipped this one initially because *Volk* is a valid German last name. Given that it's supposed to come from *Wolf*, however, it's incorrect, as it is a direct translation from *wolf* (as in the animal). It should be `Вульф` or `Вольф`, with my (albeit limited) research yielding preference to the former ([Эрик Вульф](https://ru.wikipedia.org/wiki/%D0%92%D1%83%D0%BB%D1%8C%D1%84,_%D0%AD%D1%80%D0%B8%D0%BA) and [Дик Вульф](https://ru.wikipedia.org/wiki/%D0%92%D1%83%D0%BB%D1%8C%D1%84,_%D0%94%D0%B8%D0%BA) vs. [Эмиль Вольф](https://ru.wikipedia.org/wiki/%D0%92%D0%BE%D0%BB%D1%8C%D1%84,_%D0%AD%D0%BC%D0%B8%D0%BB%D1%8C)).